utils/utils.go

```diff
@@ -464,3 +466,71 @@ func (bps *Sha256Summary) SetSha256(sha256 string) *Sha256Summary {
 	bps.sha256 = sha256
 	return bps
 }
+
+// Represents a file transfer from SourcePath to TargetPath.
+// Each of the paths can be on the local machine (full or relative) or in Artifactory (full URL).
+// File's Sha256 calculated by Artifactory during upload. we read sha256 from the HTTP's response body.
+type FileTransferDetails struct {
+	SourcePath string `json:"sourcePath,omitempty"`
+	TargetPath string `json:"targetPath,omitempty"`
+	Sha256     string `json:"sha256,omitempty"`
+}
+
+// Represent deployed artifact's details returned from build-info project for maven and gradle.
```

Comment:
Represent --> Represents...
...from build-info project... --> ...from the maven and gradle build-info extractor components.